chord_metadata_service/chord/ingest/phenopackets.py

```diff
@@ -251,22 +312,15 @@ def ingest_phenopacket(phenopacket_data: dict[str, Any], table_id: str, validate
     # Get or create all biosamples in the phenopacket
     biosamples_db = [get_or_create_biosample(bs) for bs in biosamples]
 
-    # Get or create all genes in the phenopacket
-    # TODO: May want to augment alternate_ids
-    genes_db = [get_or_create_gene(g) for g in genes]
-
-    # Get or create all diseases in the phenopacket
-    diseases_db = [get_or_create_disease(disease) for disease in diseases]
-
-    # Get or create all manually-specified HTS files in the phenopacket
-    hts_files_db = [get_or_create_hts_file(hts_file) for hts_file in hts_files]
-
     # Get or create all resources (ontologies, etc.) in the phenopacket
     resources_db = [ingest_resource(rs) for rs in resources]
 
+    interpretations_db = [get_or_create_interpretation(interp) for interp in interpretations]
+    diseases_db = [get_or_create_disease(disease) for disease in diseases]
+
     # Create phenopacket metadata object
     meta_data_obj = pm.MetaData(
-        created_by=meta_data["created_by"],
+        created_by=meta_data.get("created_by"),
```

Comment:
whats the reason for making this optional?

Reply:
My mistake, field is indeed mandatory, also changed the schema version.